src/snackbar/README.md

```diff
@@ -0,0 +1,64 @@
+# Snackbar
+
+## Introduction
+
+The Material Design Lite (MDL) **snackbar** component is a container used to notify a user of an operation's status. It displays at the bottom of the screen.
+
+Snackbars may also be used as a **toast** component. A toast may not be removed by the user and does not contain any actions.
+
+## Usage
+
+&nbsp;1. Add the `mdl-js-snackbar` class to the body.
+
+```html
+<body class="mdl-js-snackbar">
+</body>
+```
+
+&nbsp;2. Use JavaScript to trigger a snackbar.
+
+```JavaScript
+var data = {
+  message: 'Message Sent'
+};
+document.body.MaterialSnackbar.showSnackbar(data);
+```
+
+## Examples
+
+### Snackbar
+
+```JavaScript
+var data = {
+  message: 'Message Sent',
+  actionHandler: function(event) {},
+  actionText: 'Undo',
+  timeout: 10000
+};
+document.body.MaterialSnackbar.showSnackbar(data);
+```
+
+### Toast
+
+```JavaScript
+document.body.MaterialSnackbar.showSnackbar({message: 'Image Uploaded'});
+```
+
+## Data Object
+
+The Snackbar components `showSnackbar` method takes an object for snackbar data.
+The table below shows the properties and their usage.
+
+| Property | Effect | Remarks | Type |
```

Comment:
you arguments are valid :tophat: 

I was simply following the work here to be consistent :dog2: 

https://github.com/google/material-design-lite/commit/599a7da9ba173b6966c6488b0df28d167e32601f